assembly/std/index.ts

```diff
@@ -258,7 +258,7 @@ export function sendMessage(
     validityStartThread,
     validityEndPeriod,
     validityEndThread,
-    maxGas,
+ rawFees,
```

Comment:
You can use `npm run fmt` to format the code and see linting issues about jsdoc in order to make the CI green.